x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/xcontent/XContentUtils.java

```diff
@@ -115,9 +115,9 @@ public static void addAuthorizationInfo(final XContentBuilder builder, final Map
                 builder.endObject();
             }
             case SERVICE_ACCOUNT -> builder.field("service_account", authenticationSubject.getUser().principal());
-            case REMOTE_ACCESS -> {
-                // TODO handle remote access authentication
-                final String message = "remote access authentication is not yet supported";
+            case CROSS_CLUSTER_ACCESS -> {
```

Comment:
Note to self: do we still need to do something here?

Reply:
Thanks for the reminder. I created a "Misc" Jira issue to group track small things like this.